[2023-05-10T05:02:27.113+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T04:40:00+00:00 [queued]>
[2023-05-10T05:02:27.125+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T04:40:00+00:00 [queued]>
[2023-05-10T05:02:27.132+0000] {taskinstance.py:1331} INFO - Starting attempt 5 of 6
[2023-05-10T05:02:27.151+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): run_remote_script> on 2023-05-10 04:40:00+00:00
[2023-05-10T05:02:27.162+0000] {standard_task_runner.py:57} INFO - Started process 926 to run task
[2023-05-10T05:02:27.168+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'testing_stuff', 'run_remote_script', 'scheduled__2023-05-10T04:40:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmphehq1kof']
[2023-05-10T05:02:27.172+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask run_remote_script
[2023-05-10T05:02:27.266+0000] {task_command.py:410} INFO - Running <TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T04:40:00+00:00 [running]> on host 448394b5800f
[2023-05-10T05:02:27.380+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='testing_stuff' AIRFLOW_CTX_TASK_ID='run_remote_script' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T04:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T04:40:00+00:00'
[2023-05-10T05:02:27.383+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-05-10T05:02:27.385+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '#!/bin/bash\n\ncd ~\npwd']
[2023-05-10T05:02:27.401+0000] {subprocess.py:86} INFO - Output:
[2023-05-10T05:02:27.406+0000] {subprocess.py:93} INFO - /home/***
[2023-05-10T05:02:27.407+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-05-10T05:02:27.456+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=testing_stuff, task_id=run_remote_script, execution_date=20230510T044000, start_date=20230510T050227, end_date=20230510T050227
[2023-05-10T05:02:27.503+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-10T05:02:27.528+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
